SpatialGDK/Source/SpatialGDK/Private/Interop/WellKnownEntitySystem.cpp

```diff
@@ -130,4 +135,50 @@ void WellKnownEntitySystem::InitializeVirtualWorkerTranslationManager()
 	VirtualWorkerTranslationManager = MakeUnique<SpatialVirtualWorkerTranslationManager>(Receiver, Connection, VirtualWorkerTranslator);
 	VirtualWorkerTranslationManager->SetNumberOfVirtualWorkers(NumberOfWorkers);
 }
+
+void WellKnownEntitySystem::MaybeClaimSnapshotPartition()
+{
+	// Perform a naive leader election where we wait for the correct number of server workers to be present in the deployment, and then
+	// whichever server has the lowest server worker entity id claims the snapshot partition.
+	Worker_EntityId LocalServerWorkerEntityId = GlobalStateManager->GetLocalServerWorkerEntityId();
+	Worker_EntityId LowestEntityId = SpatialConstants::INVALID_ENTITY_ID;
+
+	if (LocalServerWorkerEntityId == SpatialConstants::INVALID_ENTITY_ID)
```

Comment:
Slight preference for checking this before initializing `LowestEntityId` and then initializing it to `LocalServerWorkerEntityId`.